handbook/engineering/releases/patch_release_issue_template.md

```diff
@@ -10,33 +10,57 @@ Arguments:
 
 # $MAJOR.$MINOR.$PATCH Patch Release
 
-**Attention developers:** Add pending changes to this checklist, cherry-pick the relevant commits onto branch `$MAJOR.$MINOR`. **Only check off items if the relevant PR/commits have been cherry-picked into the branch**:
+**Attention developers:** Add commits on `main` to include in this patch release to this checklist.
```

Comment:
Updated this to require patches to be in `main` (presumably meaning CI passing) before it can be indicated for cherry-picking into the patch